etl/steps/data/grapher/un/2024-07-08/maternal_mortality.py

```diff
@@ -0,0 +1,40 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("maternal_mortality")
+
+    # Read table from garden dataset.
+    tb = ds_garden["maternal_mortality"]
+
+    # only include columns not in long run dataset data://garden/maternal_mortality/2024-07-08/maternal_mortality.py
```

Comment:
I think you should keep all the columns here and preferentially use the UN MMEIG data for most of the maternal mortality charts, excluding the top two in [this dataset](https://admin.owid.io/admin/datasets/522).

The long-run variable is really good and important to have, but I think because we make some assumptions and combine the data in a slightly complicated way, it's better to keep the UN MMEIG-only variable as the primary one used in most charts. 

Could you also change the bottom two charts in [this dataset](https://admin.owid.io/admin/datasets/522) to the UN MMEIG-only variable?